hs_discover/src/components/Search.vue

```diff
@@ -184,24 +183,41 @@
             </div>
         </div>
         <div id="resource-rows" class="col-lg-9">
-            <br/>
-            <div class="table-wrapper">
-              <p class="table-message" style="color:red" v-if="(!resources.length) && (authorFilter.length ||
-              ownerFilter.length || subjectFilter.length || contributorFilter.length || typeFilter.length ||
-              availabilityFilter.length || searchtext !== '' || startdate !== '' || enddate !== '')"><i>No resource matches</i></p>
+          <div class="row" style="padding-right:14px">
+
```

Comment:
place paging controls above and below the resource table; controls will float below the filters in mobile/reduced window size use case